alibi_detect/cd/cvm.py

```diff
@@ -88,3 +95,15 @@ def feature_score(self, x_ref: np.ndarray, x: np.ndarray) -> Tuple[np.ndarray, n
             result = cramervonmises_2samp(x_ref[:, f], x[:, f], method='auto')
             p_val[f], dist[f] = result.pvalue, result.statistic
         return p_val, dist
+
+    def get_config(self) -> dict:
+        """
+        Get the detector's configuration dictionary.
+
+        Returns
+        -------
+        The detector's configuration dictionary.
+        """
+        cfg = super().get_config()
```

Comment:
The `get_config` is done at a number of levels. The mixin class does the top-level drift detector generic stuff i.e. adds `version` and `x_ref`, then most of the attributes are added in the more specific subclasses i.e. `BaseUnivariateDrift.get_config()`. Then any final detector-specific bits are added in the detector-specific subclass method. There is nothing specific for `CVMDrift`, but as an example, `FETDrift` adds in the `alternative` field:

https://github.com/SeldonIO/alibi-detect/blob/9eb5ace11967fcc7af02afdbca60f9553096a223/alibi_detect/cd/fet.py#L119

This can't be added in `BaseUnivariateDrift` since not all univariate drift detectors have an `alternative` kwarg. 

P.s. The use of this of this mixin is probably a good argument for having a drift detector base class (in between `BaseDetector` and `BaseUnivariateDrift` etc). However, I didn't want to change the core of the code too much in this PR...